src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -11,28 +11,54 @@
 import frc.robot.Constants;
 
 public class IntakeSubsystem extends SubsystemBase {
-
   private CANSparkMax intakeMotor;
-
+  private CANSparkMax chooserMotor;
   public IntakeSubsystem() {
     intakeMotor = new CANSparkMax(Constants.IntakeConstants.INTAKE_MOTOR_ID, MotorType.kBrushless);
+    chooserMotor =
+        new CANSparkMax(Constants.IntakeConstants.INTAKE_CHOOSER_ID, MotorType.kBrushless);
     intakeMotor.restoreFactoryDefaults();
+    chooserMotor.restoreFactoryDefaults();
     intakeMotor.setSmartCurrentLimit(40);
+    chooserMotor.setSmartCurrentLimit(20);
+  }
+
+  // TODO: Add check to make sure turret is below 45 degrees before running & add photogate when
+  // implemented.
+  public void trapIntake() {
+    reverseChooser();
+    runIntake();
   }
 
-  public Command intakeCommand() {
-    // Inline construction of command goes here.
-    // Subsystem::RunOnce implicitly requires `this` subsystem.
-    return runOnce(
-        () -> {
-          /* one-time action goes here */
-        });
+  public void runIntake() {
+    intakeMotor.set(Constants.IntakeConstants.INTAKE_PERCENTAGE);
   }
 
-  public void setIntakeSpeed(double speed) {
-    intakeMotor.set(speed);
+  public void reverseIntake() {
+    intakeMotor.set(-Constants.IntakeConstants.INTAKE_PERCENTAGE);
   }
 
+  public void setChooseSpeed() {
+    chooserMotor.set(Constants.IntakeConstants.INTAKE_PERCENTAGE);
+  }
+
+  public void reverseChooser() {
+    chooserMotor.set(-Constants.IntakeConstants.INTAKE_PERCENTAGE);
+  }
+
+  public void stopIntake() {
+    intakeMotor.stopMotor();
+    chooserMotor.stopMotor();
+  }
+
+  public void chooserState() {
```

Comment:
A method name should be a verb. This does not describe whatever action this is meant to perform